.github/workflows/release-release-events-proxy.yml

```
@@ -0,0 +1,14 @@
+name: 'Release: Release events proxy workflow'
+description: 'Proxy workflow to dispatch workflows from trunk branch for release events (published and pre-release).'
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)